src/features/common/Layout/ProjectsLayout/MobileProjectsLayout.tsx

```diff
@@ -1,31 +1,32 @@
-import { FC, useState } from 'react';
-import style from './ProjectsLayout.module.scss';
+import { FC } from 'react';
+import styles from './ProjectsLayout.module.scss';
 import ProjectsMap from '../../../projectsV2/ProjectsMap';
-import WebappButton from '../../WebappButton';
 import { SetState } from '../../types/common';
 import { ProjectsProvider } from '../../../projectsV2/ProjectsContext';
 import { ProjectsMapProvider } from '../../../projectsV2/ProjectsMapContext';
+import Credits from '../../../projects/components/maps/Credits';
+import { ViewMode } from '../../../../../pages/_app';
 
 interface ProjectsLayoutProps {
   currencyCode: string;
   setCurrencyCode: SetState<string>;
   page: 'project-list' | 'project-details';
+  selectedMode: ViewMode;
+  setSelectedMode: SetState<ViewMode>;
+  isMobile: boolean;
 }
 
 const MobileProjectsLayout: FC<ProjectsLayoutProps> = ({
   children,
   page,
   currencyCode,
   setCurrencyCode,
+  selectedMode,
+  setSelectedMode,
+  isMobile,
 }) => {
-  const [isMapMode, setIsMapMode] = useState(false);
-
-  const mobileLayoutClass = `${style.mobileProjectsLayout} ${
-    isMapMode ? style.mapMode : ''
-  }`;
```

Comment:
Why did this state need to be moved to the _app.tsx file, when it is not needed for both desktop and mobile layouts?

Reply:
removed it from the _app.tsx and  placed it  into the `MobileProjectsLayout.tsx`